modules/nf-core/last/train/main.nf

```diff
@@ -31,6 +32,28 @@ process LAST_TRAIN {
         $fastx \\
         > ${prefix}.\$INDEX_NAME.train
 
+    echo "# id: 'alignment parameters'" > ${prefix}_mqc.tsv
+    echo "# section_name: 'Alignment parameters and summary'" >> ${prefix}_mqc.tsv
+    echo "# format: 'tsv'" >> ${prefix}_mqc.tsv
+    echo "# plot_type: 'table'" >> ${prefix}_mqc.tsv
+    echo "# description: 'This plot shows the last alignment parameters'" >> ${prefix}_mqc.tsv
+    echo "# pconfig:" >> ${prefix}_mqc.tsv
+    echo "#    id: 'alingment parameters'" >> ${prefix}_mqc.tsv
+    echo "#    title: 'alingment parameters'" >> ${prefix}_mqc.tsv
+    echo "#    ylab: ''" >> ${prefix}_mqc.tsv
+    echo "id\tsubstitution_percent_identity\tlast -t\tlast -a\tlast -A\tlast -b\tlast -B\tlast -S" >> ${prefix}_mqc.tsv
+    for i in *.\$INDEX_NAME.train
```

Comment:
There will only be a single `.train` file in the work directory of the `last/train` module, so you do not need a `for` loop.